draft-ietf-tls-ctls.md

```diff
@@ -419,7 +423,7 @@ transports length information may be inferred from the underlying layer.
 
 Normal DTLS does not provide a mechanism for suppressing the sequence number
 field entirely.  In cases where a sequence number is not required (e.g., when a
-reliable transport is in use), a cTLS implementation may suppress it by setting
+reliable transport is in use), a cTLS implementation MUST suppress it by setting
```

Comment:
This strikes me as a bit odd.  Why not just make it automatic for streaming transports?

Conversely, is it ever reasonable to set this flag on a non-streaming transport?  Or could we just infer this setting from the transport and skip the configuration entirely?